script/release/README.md

```diff
@@ -192,6 +192,8 @@ be handled manually by the operator:
     - Bump the version in `compose/__init__.py` to the *next* minor version
       number with `dev` appended. For example, if you just released `1.4.0`,
       update it to `1.5.0dev`
+    - Update compose_version in [](https://github.com/docker/docker.github.io/blob/master/_config.yml) and [https://github.com/docker/docker.github.io/blob/master/_config_authoring.yml](https://github.com/docker/docker.github.io/blob/master/_config_authoring.yml)
```

Comment:
[] needs to have some text in it

Reply:
[https://github.com/docker/docker.github.io/blob/master/_config_authoring.yml] could remove the https:// here to match what you do lower down

Reply:
Done